doc/news/changes.h

```diff
@@ -120,13 +120,25 @@ inconvenience this causes.
 <h3>General</h3>
 
 <ol>
+
+ <li> New: Add new classes to expand scalar finite element solution into
+ orthogonal bases FESeries::Fourier and FESeries::Legendre. Also
```

Comment:
I would add two articles to the first sentence: "Add new classes to expand a scalar finite element solution into the orthogonal bases FESeries::Fourier and FESeries::Legendre"